frontend/public/components/network-policy.jsx

```diff
@@ -199,7 +199,7 @@ const Details_ = ({ obj: np, flags }) => {
         <SectionHeading text={t('network-policy~Ingress rules')} />
         <p className="co-m-pane__explanation">
           {t(
-            'network-policy~Pods accept all traffic by default. They can be isolated via NetworkPolicies which specify a whitelist of ingress rules. When a Pod is selected by a NetworkPolicy, it will reject all traffic not explicitly allowed via a NetworkPolicy. See more details in:',
+            'network-policy~Pods accept all traffic by default. They can be isolated via NetworkPolicies which specify a whitelist of ingress rules. When a Pod is selected by a NetworkPolicy, it will reject all traffic not explicitly allowed via a NetworkPolicy. See more details in: ',
```

Comment:
This should actually be using the `Trans` component so that the link is part of the message and can be translated.

https://react.i18next.com/latest/trans-component

Here's an example:

https://github.com/kdoberst/console/blob/48c7c0a23723bdf2b7fe66283558d16f2d889e58/frontend/public/components/monitoring/receiver-forms/routing-labels-editor.tsx#L97-L105

Reply:
Sure I'll look into it